.github/workflows/reusable-web-deploy.yml

```diff
@@ -87,6 +87,8 @@ jobs:
           echo REACT_APP_STRIPE_CLIENT_KEY=${{ secrets.STRIPE_CLIENT_KEY }} >> .env
           echo REACT_APP_NOVU_GTM_ID=${{ secrets.REACT_APP_NOVU_GTM_ID }} >> .env
           echo REACT_APP_MIXPANEL_KEY=${{ secrets.MIXPANEL_TOKEN }} >> .env
+          echo REACT_APP_CLERK_PUBLISHABLE_KEY=${{ secrets.REACT_APP_CLERK_PUBLISHABLE_KEY }} >> .env
+          echo REACT_APP_IS_EE_AUTH_ENABLED=${{ secrets.REACT_APP_IS_EE_AUTH_ENABLED }} >> .env
```

Comment:
✏ **suggestion:** Please call the secrets without the `REACT_APP_` prefix. Like:
```suggestion
          echo REACT_APP_CLERK_PUBLISHABLE_KEY=${{ secrets.CLERK_PUBLISHABLE_KEY }} >> .env
          echo REACT_APP_IS_EE_AUTH_ENABLED=${{ secrets.IS_EE_AUTH_ENABLED }} >> .env
```